### DISCUSSION TITLE
Is there a good way to insert edges in a graph instance?

### DISCUSSION THREAD

**Discussion by bryanzhou008** [pyg-team/pytorch_geometric#4614] [Q&A]
I am puzzled as I couldn't find any online answers explaining this. Which leads me to wonder if this data type is not meant to be operated on in this manner?

**Comment by rusty1s** [comment]
You can insert edged by concatenating them to your `edge_index`:
```python
new_edges = torch.tensor([[0, 1, 2, 3], [1, 0, 3, 2]])
edge_index = torch.cat([edge_index, new_edges], dim=1)
```
